### DISCUSSION TITLE
Where do hex and bin files go when you Flash a Daisy?

### DISCUSSION THREAD

**Discussion by TricksterSam** [Wasted-Audio/hvcc#165] [Q&A] [answered]
I'm using PopOS, and I can't find the hex file that hvcc creates when I compile and flash with PlugData? I'd like to be able to test this file and make it available to other people to use.

**Comment by dromer** [comment, marked as answer]
When you flash directly from the program everything is done in a temporary directory in `/tmp`.

If you want to export a binary use the .. wait for it .. Binary export type!

**Comment by TricksterSam** [comment]
I feel sheepish now. Thank you for taking the time to point out what should have been obvious!
